phpBB/styles/prosilver/template/bbcode.html

```diff
@@ -8,6 +8,22 @@
 <!-- BEGIN listitem --><li><!-- END listitem -->
 <!-- BEGIN listitem_close --></li><!-- END listitem_close -->
 
+<!-- BEGIN mention -->
```

Comment:
is this temp code? the xsl stuff needs to be extrapolated out into the core. we should not be shipping any xsl in the bbcode.html file. Joshy needs to remove what he left there...

Reply:
I wasn't aware of Joshy's plans for that and this place seemed to be the right.
Moved to the core.

There are also many legacy stuff like first and second passes I suppose.
Sometimes it is not obvious to distinguish the legacy, and I suppose that new BBCodes should not be handled by that code.